source/dataman/MdtmMan.cpp

```diff
@@ -0,0 +1,250 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ *
+ * MdtmMan.cpp
+ *
+ *  Created on: Apr 20, 2017
+ *      Author: Jason Wang
+ */
+
+#include "MdtmMan.h"
+
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "zmq.h"
```

Comment:
Use `<zmq.h>` instead of `"zmq.h"` since `zmq.h` is known to be an external header.  Only use `"foo.h"` when `foo.h` is known to be part of this project.